ui/app/adapters/application.js

```diff
@@ -44,7 +44,9 @@ export default class ApplicationAdapter extends RESTAdapter {
     return super.findAll(...arguments).catch((error) => {
       const errorCodes = codesForError(error);
 
-      const isNotImplemented = errorCodes.includes('501');
+      const isNotImplemented =
+        errorCodes.includes('501') ||
+        error.message.includes("rpc: can't find service");
```

Comment:
Note for reviewers: this error comes from the stdlib: https://cs.opensource.google/go/go/+/refs/tags/go1.20.6:src/net/rpc/server.go;l=613

Reply:
There's a change a UI dev* might short-circuit this with an isNotImplemented handler in the future that just checks for `501` in a specific adapter layer. I don't think it's likely (we've always used `super` so far to inherit from this one) but it would be safer if we eventually returned 501s for these. I don't know if changing the stdlib is at all a likelihood, though!

*it could be me! I forget stuff all the time!

Reply:
We would need to wrap the RPC error to change the status code. Not a big problem, but I suspect there may be lots of code to change 😅 

I opened https://github.com/hashicorp/nomad/issues/18050 to track this.